include/deal.II/base/vector_slice.h

```diff
@@ -60,59 +61,62 @@ class VectorSlice
    * The real constructor for a vector slice, allowing you to specify the
    * start index and the length of the slice.
    */
-  VectorSlice(VectorType   &v,
-              unsigned int start,
-              unsigned int length);
+  VectorSlice(VectorType &v, unsigned int start, unsigned int length);
```

Comment:
Hm, this is not our common style, though...

Reply:
But does it enforce merging it to one line? In other words, what changed this from our usual style to the way shown here?

Reply:
`clang-format` just sees a parameter list. There is no representation for the line each parameter was on. For `clang-format` (unless specified by further options) breaking any line comes with a penalty. Hence, not breaking is considered best.

There is an experimental option `ExperimentalAutoDetectBinPacking` that tries to address exactly this problem. However, it doesn't seem to work here properly, i.e. there is no difference between `ExperimentalAutoDetectBinPacking: false` and `ExperimentalAutoDetectBinPacking: true`.

Reply:
I'm ok with it if clang-format doesn't break this by itself. I'm not ok with it if clang-format re-merges these lines by itself.